data/kahet/kahet missions.txt

```diff
@@ -303,6 +303,574 @@ mission "Ringworld Debris: Quarg"
+			`	Despite the sign language, you notice that he seemed strikingly halfhearted compared to the previous times you have encountered him. As he heads toward his Gull, the Late Season Dancing, you can't help but wonder why.`
+				decline
+
+mission "Ka'het: Death in the Graveyard 3"
+	landing
+	name "Angry Ka'het"
+	description "Destroy the Telis'het in <system> to prevent it from destroying the Ka'het on the surface."
+	to offer
+		has "Ka'het: Death in the Graveyard 1: done"
+	on offer
+		conversation
+			`It's daytime now. Flying in formation with the Late Season Dancing, you spot the Maeri'het on the ground easily: it has spread a pair of enormous solar wings, at least thrice as long as the body of the Ka'het itself. A look at your radar reveals more, however. There is a second Ka'het standing above the downed one, classified by your IFF as a Telis.`
+			`	"It is trying to reanimate the Maeri'het," Plume chants over the intercom, "but I do not think it can do so. It has been too long since you found it for repairs to have any effect." Among the smoke clouds you distinguish pairs of tiny arms protrude from the bottom of the Telis'het, pressing all over the corpse below it. You record some electric shocks move between the two, but the Maeri'het does not react.`
+			choice
+				`	"Let's wait for it to leave then."`
+				`	"Should I engage it?"`
+					goto engage
+			`	"Ah, it is not that easy. Its programming may be telling it to heal it, but the slug will grow more frustrated with each failed attempt. You see, one of the functions left under the control of the biological side is the combat. It may not be able to decide to stop trying to heal it, but it can decide to register it as a foe, destroy it, and then leave freely as if it was never there," he concludes with a cold chant.`
+				goto firepower
+			label engage
+			`	"Ah, I am afraid so. Its programming may be telling it to heal it, but the slug will grow more frustrated with each failed attempt. You see, one of the functions left under the control of the biological side is the combat. It may not be able to decide to stop trying to heal it, but it can decide to register it as a foe, destroy it, and then leave freely as if it was never there," he concludes with a cold chant.`
+			label firepower
+			`	"And thus, the Telis is a threat to the Remnant xenobiological interests. If you have enough firepower to do so, I ask you to attack the Ka'het and lead it away, so that we can loot the carcass while it is still useful."`
```

Comment:
```suggestion
			`	"And thus, the Telis is a threat to the Remnant's xenobiological interests. If you have enough firepower to do so, I ask you to attack the Ka'het and lead it away, so that we can loot the carcass while it is still useful."`
```